### ISSUE TITLE
Add in doc strings

### ISSUE THREAD

**Issue by nneveu** [slaclab/lcls-tools#91]
Problem: Some classes/functions do not have doc strings.
Solution: At least a one line comment/doc string for all classes/functions.
Alternatives: What we have now, mixed or no documentation for some classes/functions.

**Comment by nneveu** [comment]
@lisazacarias can you add an .rst file for any sc code you want to doc? https://github.com/slaclab/lcls-tools/tree/main/docs/source

After there's an rst, the github action will generate the doc w/ new PR's by auto grabbing any doc strings you have.

**Comment by lisazacarias** [comment]
Yeah, is there documentation somewhere with how to write one? I tried googling and looking at the files already there but it's still kinda cryptic

**Comment by nneveu** [comment]
Yea, it's Sphinx: https://www.sphinx-doc.org/en/master/usage/quickstart.html
I can show you if you still have questions.
